beater/config/sampling.go

```diff
@@ -71,15 +72,26 @@ type TailSamplingPolicy struct {
 }
 
 func (c *TailSamplingConfig) Unpack(in *config.C) error {
+	var err error
+	defer func() {
+		if err != nil {
+			logger := logp.NewLogger(logs.Config)
+			logger.Errorf("failed to setup tail sampling: %v", err)
+			logger.Info("continuing with tail sampling disabled")
```

Comment:
happy to change these messages to whatever the reviewer things is appropriate. also, i felt like there should be an additional log message about running with tbs disabled, but for some reason i thought it should be `info` level. happy to change or discuss this, as I don't have any well-formulated reason for choosing `info`.